### ISSUE TITLE
Default results list for report

### ISSUE THREAD

**Issue by arnoudplantinga** [WillemSleegers/tidystats-v0.3#61]
Would it be weird to have a default for the results list in the `report` function? It's annoying to start with `report(results, ...)` everytime I want to report something.

**Comment by WillemSleegers** [comment]
I've been playing with that idea as well. It's a bit weird, but it might be worth it. 

I've already implemented a version of this using options(). If you set 'tidystats_list' in `options()` to the name of your results list, then you do not need to supply it to the `report()` function anymore.

For example: `report("M1_1")` results in: *t*(199) = 24.22, *p* < .001

I did have to move the results argument in `report()` to the back of all the arguments, so if you want to explicitly set it, it will be last in line (which is not a big deal I think).
